website/docs/maintenance/backup_restore.md

```diff
@@ -8,18 +8,18 @@ If you are using `docker-compose`, you are using Docker Compose v1, which has be
 
 ## Backup
 
-Create backup file `teslamate.bck`:
+Create backup file:
 
 ```bash
-docker compose exec -T database pg_dump -U teslamate teslamate > ./teslamate.bck
+docker compose exec -T database pg_dump -U teslamate teslamate > teslamate_database_$(date +%F_%H-%M-%S).sql
```

Comment:
What about using only a date (2025-05-16 here) - that way copy and paste of commands should still work (as users are most likely updating on the same day).

Reply:
Sorry, not entirely sure what you are asking for?

Reply:
going with `$(date +%Y-%m-%d)` and not using %H-%M-%S will allow the restore command to be a copy / paste without users manually searching for the exact filename taken earlier (assuming that most upgrades happen at the same day).

Reply:
Good question.

Was trying to avoid the situation here where you might have a good backup, but then you accidentally run the above command again after the database is shutdown/replaced and replace the good backup with a bad backup. And not notice the glaring error message saying it failed.

Yes, would like to think this wouldn't happen, but we did get a report from somebody who had a 0 byte backup file, and this is one way that could occur.

At least with the above, there is some increased chance you might still have the good backup file.